codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/JsonRpcProtocolGenerator.java

```diff
@@ -63,10 +63,13 @@ protected void serializeInputDocument(GenerationContext context, OperationShape
         GoWriter writer = context.getWriter();
         StructureShape input = ProtocolUtils.expectInput(context.getModel(), operation);
         String functionName = ProtocolGenerator.getDocumentSerializerFunctionName(input, getProtocolName());
-        writer.addUseImports(SmithyGoDependency.SMITHY_JSON);
 
-        // If there are no members then there's nothing to serialize
-        if (input.members().size() == 0) {
```

Comment:
We might still need this check. Why generate serializers if no input members?

Reply:
Will update PR shortly that addresses this issue.